plugins/inputs/jti_openconfig_telemetry/README.md

```diff
@@ -0,0 +1,46 @@
+# JTI OpenConfig Telemetry Input Plugin
+
+This plugin reads Juniper Networks implementation of OpenConfig telemetry data from listed sensors using Junos Telemetry Interface. Refer to
+[openconfig.net](http://openconfig.net/) for more details about OpenConfig and [Junos Telemetry Interface (JTI)](https://www.juniper.net/documentation/en_US/junos/topics/concept/junos-telemetry-interface-oveview.html).
+
+### Configuration:
+
+```toml
+# Subscribe and receive OpenConfig Telemetry data using JTI
+[[inputs.jti_openconfig_telemetry]]
+  server = ["localhost:1883"]
+
+  ## Frequency to get data in milliseconds
+  sampleFrequency = 2000
```

Comment:
Make this an `internal.Duration` so it can be specified like: `"2000ms"`

Reply:
Using internal.Duration for sampleRrequency/reportingRate